### ISSUE TITLE
Trial Balance with Dimension - MDTL UK Tenant-CAS-01410-N2X7K1

### ISSUE THREAD

**Issue by Prakash3K** [3ktech/General#656]
**Approved time 3 Hours**

**Trail Balance Custmized (50107, Report Request)**

[Trial Balance With Dimension.xlsx](https://github.com/3ktech/General/files/12461113/Trial.Balance.With.Dimension.xlsx)

**Comment by TabrejAlam3K** [comment]
Mr. @3ktech - Can you please share the time effort for this

We have already done the same things for India BC Tenant

CC : @audhi3k @Shasthara @narayana3kt

**Comment by TabrejAlam3K** [comment]
Hi Mr. @3ktech - Please assign the technical who will work upon.

CC : @audhi3k

**Comment by audhi3k** [comment]
Mr Karthik @3ktech share the time estimate and align the case to technical

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Please update the time effort first before starting the work.

CC : @audhi3k  @3ktech

**Comment by sruthi1017** [comment]
i am having the old password it is not allowing to sign in. can u please provide me the latest password @TabrejAlam3K 
CC: @3ktech

**Comment by TabrejAlam3K** [comment]
User will provide the new credential to logon into the BC.

I was connect with Mr. Boopathy, He will provide the Password soon.

CC : @audhi3k @sruthi1017

**Comment by TabrejAlam3K** [comment]
We are still waiting for Credential and Approval for the same.

CC : @audhi3k

**Comment by TabrejAlam3K** [comment]
Approval received from the client side, Still waiting for login credentials.

CC : @audhi3k @sruthi1017

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Not able to find anything related to the content of Trial Balance with Dimension

![Image](https://github.com/3ktech/General/assets/116171982/33ac9fe8-3004-4788-97b9-5683d928ec17)

CC : @audhi3k

**Comment by sruthi1017** [comment]
hi @TabrejAlam3K it is in the sandbox of UATProd

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - It is not there I am not able to find the Report in Merit UK Sandbox Tenant. Please help us to fix it as on priority.

CC : @audhi3k @3ktech

**Comment by 3ktech** [comment]
@sruthi1017 , connect with @TabrejAlam3K  and sort this out.

**Comment by sruthi1017** [comment]
hi @TabrejAlam3K can we connect for this i can show u where the report i have been deployed.

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - As we discussed in Morning have you deployed in the Sandbox Environment (Merit Data and Technology).

CC : @audhi3k @sandhya3k

**Comment by sruthi1017** [comment]
hi @TabrejAlam3K it is published in sandbox

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Make sure this changes should be reflecting in the Sandbox under the Merit Data & Technology Limited Company.

CC : @audhi3k @3ktech

**Comment by TabrejAlam3K** [comment]
![Image](https://github.com/3ktech/General/assets/116171982/b2ddebd4-436c-4503-878f-370ee31de86a)

**Comment by TabrejAlam3K** [comment]
.

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Please deployed this into the Production and confirm. Report is working fine in the Sandbox Environment.

![Image](https://github.com/3ktech/General/assets/116171982/c8f55bf9-bd6c-43ca-be0c-157f29612730)

CC : @audhi3k @3ktech @Shasthara

**Comment by TabrejAlam3K** [comment]
Hi Ms. @sruthi1017 - Please update you have deployed the changes into the Production.

**Comment by Jayanth3KT** [comment]
Hi @TabrejAlam3K Can I know the current status of this ticket.

CC: @Shasthara @audhi3k

**Comment by TabrejAlam3K** [comment]
Hi Mr. @Jayanth3KT - Waiting for Customer Response on.

CC : @audhi3k @Shasthara

**Comment by TabrejAlam3K** [comment]
Waiting for Customer Confirmation

CC : @audhi3k @Jayanth3KT @Shasthara

**Comment by TabrejAlam3K** [comment]
confirmation received for closure going to close this ticket.

Mr. @Jayanth3KT - Close this ticket from the support portal. (Approved time 3 Hours)

CC : @audhi3k @Shasthara
